.github/workflows/build.yml

```diff
@@ -0,0 +1,10 @@
+name: build
+
+on:
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  test:
```

Comment:
name this to `tox` rather than `test` because its build, lint, and run unit test.